.github/workflows/build-images.yaml

```diff
@@ -125,6 +127,74 @@ jobs:
         env:
           LAST_GREEN_COMMIT: ${{ needs.determine-last-green-commit.outputs.lastGreenCommit }}
 
+  # TODO(rustielin): build these images continuously rather than on PR
+  rust-images-testing:
```

Comment:
Can we abstract the image build into a sub workflow and call it twice here?

Then we can actually use it more simply elsewhere too